README.md

```diff
@@ -1,13 +1,8 @@
+# PLEASE BE ADVISED
+This branch is the main branch for 6.0.0 development. It is nearing an alpha release and while not considered 
```

Comment:
Since all branches will have this message, can I suggest the following change of phrasing:
```
The `main` branch of Fedora contains the most current state of 6.0.0 development. It is nearing...
```